cnf-certification-test/accesscontrol/resources/resources.go

```diff
@@ -40,3 +40,38 @@ func HasRequestsAndLimitsSet(cut *provider.Container) bool {
 	}
 	return passed
 }
+
+// For more info on cpu mgmt polcies see https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/.
+func HasExclusiveCPUsAssigned(cut *provider.Container) bool {
+	cpuLimits := cut.Resources.Limits.Cpu()
+	cpuRequests := cut.Resources.Requests.Cpu()
```

Comment:
LGTM, can Line 47 be put later after Line 57, when you are checking on cpuRequests?

Reply:
Done.